index.html

```diff
@@ -1,72 +1,225 @@
+                    <li>Limited Support</li>
+                </ul> -->
+                <a href="https://sgrewritten.org/wiki" class="btn my-4 font-weight-bold atlas-cta cta-ghost">Join our Discord</a>
+            </div>
+            <div data-aos="fade-left" data-aos-delay="200" data-aos-duration="1000" data-aos-once="true" class="col-md-4 text-center py-4 mt-5">
+                <h4 class="my-4">Developers</h4>
+                <!-- <p class="font-weight-bold">$ <span class="display-2 font-weight-bold">0</span> / MO.</p>
+                <ul class="list-unstyled">
+                    <li>Unlimited Documents</li>
+                    <li>Unlimited Reviews</li>
+                    <li>Unlimited Members</li>
+                    <li>Unlimited Support</li>
+                </ul> -->
+                <a href="https://sgrewritten.org/codestyle" class="btn my-4 font-weight-bold atlas-cta cta-ghost">Join our Discord</a>
+            </div>
+            <div data-aos="fade-right" data-aos-delay="200" data-aos-duration="1000" data-aos-once="true" class="col-md-4 text-center py-4 mt-5">
+                <h4 class="my-4">Sponsors</h4>
+               <!-- <p class="font-weight-bold">$ <span class="display-2 font-weight-bold">0</span> / MO.</p>
+                <ul class="list-unstyled">
+                    <li>Up to 5 Documents</li>
+                    <li>Up to 3 Reviews</li>
+                    <li>5 team Members</li>
+                    <li>Limited Support</li>
+                </ul> -->
+                <a href="https://sgrewritten.org/sponsor" class="btn my-4 font-weight-bold atlas-cta cta-ghost">Sponsor</a>
+            </div>
+            <div data-aos="fade-left" data-aos-delay="200" data-aos-duration="1000" data-aos-once="true" class="col-md-4 text-center py-4 mt-5">
+                <h4 class="my-4">Alpha Testers</h4>
+                <!-- <p class="font-weight-bold">$ <span class="display-2 font-weight-bold">0</span> / MO.</p>
+                <ul class="list-unstyled">
+                    <li>Up to 15 Documents</li>
+                    <li>Up to 10 Reviews</li>
+                    <li>25 team Members</li>
+                    <li>Limited Support</li>
+                </ul> -->
+                <a href="https://discord.com/channels/853659042799026177/853659045671337996/978737916782211162" class="btn my-4 font-weight-bold atlas-cta cta-ghost">Join our Discord</a>
+            </div>
+        </div>
+    </div>
+    <!-- contact -->
+    <div class="jumbotron jumbotron-fluid" id="contact" style="background-image: url(img/bg.jpg);">
+        <div class="container my-5">
+            <div class="row justify-content-between">
+                <div class="col-md-6 text-white">
+                    <h2 class="font-weight-bold">Contact Us</h2>
+                    <p class="my-4">
+                        "We got a message from the Tok'ra. Apparently they've been trying to contact us for over three months." <br>          

```

Comment:
I added a SG related quote, but it probably broke spacing.
Will need to look at that once the site builds again.